app/helpers/facility_reservations_helper.rb

```diff
@@ -4,8 +4,13 @@ def offline_category_collection
     I18n.t("offline_reservations.categories").invert.to_a
   end
 
-  def reservation_category_label(category)
-    I18n.t(category.presence, scope: "offline_reservations.categories", default: "")
+  def admin_category_collection
+    I18n.t("reservations.categories").invert.to_a
+  end
+
+  def reservation_category_label(reservation)
+    scope = reservation.is_a?(OfflineReservation) ? "offline_reservations.categories" : "reservations.categories"
```

Comment:
In this case, the reservation should be either an `AdminReservation` or `OfflineReservation`, correct? Can we move the locales to be under `admin_reservations` since that's really what the categories apply to and use "#{reservation.class.name.underscore}`?

I'm also wondering if we should remove the default so we'd get errors in development if we're looking at an invalid category.